### ISSUE TITLE
Issue running on Linux

### ISSUE THREAD

**Issue by synologic** [adamabdelhamed/PowerArgs#158]
When using PromptIfMissing = true on an argument and running the app on Linux, i get:

```
Unhandled exception. System.TypeInitializationException: The type initializer for 'PowerArgs.ConsoleProvider' threw an exception.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at PowerArgs.RGB.Convert(ConsoleColor& color)
   at PowerArgs.StdConsoleProvider..ctor()
   at PowerArgs.ConsoleProvider..cctor()
   --- End of inner exception stack trace ---
   at PowerArgs.RichTextCommandLineReader..ctor()
   at PowerArgs.CliHelper..ctor()
   at PowerArgs.ArgRequired.ValidateAlways(CommandLineArgument argument, String& arg)
   at PowerArgs.CommandLineArgument.Validate(String& commandLineValue)
   at PowerArgs.CommandLineArgument.Populate(HookContext context)
   at PowerArgs.CommandLineArgument.PopulateArguments(List`1 arguments, HookContext context)
   at PowerArgs.Args.ParseInternal(CommandLineArgumentsDefinition definition, String[] input)
   at PowerArgs.Args.<>c__DisplayClass15_0.<ParseAction>b__0()
   at PowerArgs.Args.Execute[T](Func`1 argsProcessingCode)
   at PowerArgs.Args.ParseAction(CommandLineArgumentsDefinition definition, String[] args)
   at PowerArgs.Args.ParseAction(Type t, String[] args)
   at PowerArgs.Args.Parse(Type t, String[] args)
   at PowerArgs.Args.Parse[T](String[] args)
   at MulticastMonitor.Main(String[] args)
Aborted
```

Please advise

**Comment by wylee** [comment]
I'm running into the same issue on macOS. It's reproducible by running the following C# snippet:

```
var color = System.Console.ForegroundColor;
PowerArgs.RGB.Convert(color);
```

It seems the underlying issue is that `System.Console.BackgroundColor` and `System.Console.ForegroundColor` aren't set on non-Windows platforms as shown in this `fsi` session:

```
> System.Console.BackgroundColor;;
val it: System.ConsoleColor = -1

> System.Console.ForegroundColor;;
val it: System.ConsoleColor = -1
```

Looking at the commit history, it appears this was [fixed recently](da7b0c8e1c280ade9c55be6ba9f023624f009001), but the fix hasn't been published. In the meantime, I found that it's possible to work around this by setting `System.Console.BackgroundColor` and `System.Console.ForegroundColor` before invoking `PowerArgs`:

```
System.Console.BackgroundColor = System.ConsoleColor.Black;
System.Console.ForegroundColor = System.ConsoleColor.White;
```

**Comment by adamabdelhamed** [comment]
This will be addressed in the version I publish today. We'll fallback to Gray and Black if we can't read the existing FB and BG from the system. Thanks for reporting.

**Comment by adamabdelhamed** [comment]
Fixed in 4.0.1
